sdk/src/settings/builder.rs

```diff
@@ -404,18 +409,21 @@ pub(crate) struct BuilderSettings {
     ///
     /// For more information on the reasoning behind this field see [ActionsSettings].
     pub actions: ActionsSettings,
-
-    // Certificate statuses will be fetched for either all the manifest labels, or just the active manifest.
-    pub certificate_status_fetch: Option<OcspFetch>,
-
-    // Whether or not existing OCSP responses should be overridden by new values.
+    // REVIEW NOTE: should this be in builder settings or core?
+    /// Certificate statuses will be fetched for either all the manifest labels, or just the active manifest.
+    pub certificate_status_fetch: Option<OcspFetchScope>,
+    /// Whether or not existing OCSP responses should be overridden by new values.
     pub certificate_status_should_override: Option<bool>,
 }
 
+/// The scope of which manifests to fetch for OCSP.
+#[cfg_attr(feature = "json_schema", derive(schemars::JsonSchema))]
 #[derive(Copy, Clone, Debug, Deserialize, PartialEq, Serialize)]
 #[serde(rename_all = "lowercase")]
-pub(crate) enum OcspFetch {
+pub enum OcspFetchScope {
```

Comment:
do we need this or can we just retrieve the active manifest ocsp?

Reply:
Hm, that's a good question. I think supporting both is a valid use case, but I'm not sure how people normally expect it to work. We should probably support OCSP fetching at a more programatic level to support these cases.